app/controllers/admin/roles_controller.rb

```diff
@@ -14,12 +16,15 @@ def index
     end
 
     def show
-      @url = if @track
-               toggle_user_track_admin_conference_role_path(@conference.short_title, @role.name, @track)
-             else
-               toggle_user_admin_conference_role_path(@conference.short_title, @role.name)
-             end
+      unless @conference.nil?
+        @url = if @track
+                 toggle_user_track_admin_conference_role_path(@conference.short_title, @role.name, @track)
+               else
+                 toggle_user_admin_conference_role_path(@conference.short_title, @role.name)
+               end
+      end
       @users = @role.users
+      render 'show_org_admins' if @conference.nil?
```

Comment:
I feel like this controller is doing too much, with too many conditionals. Is a conference loaded? Is an organization loaded? Is there a track? etc etc...
Would it make sense to move all the Org-admin related stuff to its own controller?

Other than that, the UI works great :)